project/Dependencies.scala

```diff
@@ -62,4 +62,11 @@ object Dependencies {
   lazy val exi = Seq(
     "com.siemens.ct.exi" % "exificient" % "1.0.7"
   )
+
+  // Scala Steward may try to update this version to include the Scala version,
+  // for example 0.18_2.12.15. This is incorrect because the unidoc plugin uses
+  // crossVersion to figure out the Scala version. This should be set to just the
+  // version of the genjavadoc plugin, without the Scala version.
+  lazy val genjavadoc: String = "0.19"
```

Comment:
I don't know how scala steward knows how to update this version, but based on the comment it can. But I wonder if it does it because it knows about the "unidocGenjavadocVersion" key, which is no longer right next to this version? Maybe we should change this to something like:

```scala
lazy val genjavadoc = Seq(
  unidocGenjavadocVersion := "0.19"
)
```

and

```scala
 .enablePlugins(GenJavadocPlugin)
 .settings(Dependencies.genjavadoc)
}
```
I would hope that makes it more like it can find the dependency string since it's very similar to what it was before.